src/components/CountryTemperature/index.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useState } from "react";
+import getWeatherAPI from "/src/api/weatherReport"
+import setCountryList from "../../config/countryReport"
+import WeatherImages from "../../config/enums/image"
+import CountryReport from "../../interfaces/CountryReport/index"
+// import DayW1eatherReport from "../DayWeather";
+
+//style
+import "./style.css";
+
+const CountryTemperature = () => {
+  const [searchInput,setSearchInput] = useState(() => "")
+  const [countryName, setCountryName] = useState(() => "New York");
+  const [cityName, setCityName] = useState(() => "United States");
+  const [countryCurrentWeather, setCountryCurrentWeather] = useState(
+    () => "Partly Clowdy"
+  );
+  const [countryCurrentTemperature, setCountryCurrentTemperature] = useState(
+    () => "21.0"
+  );
+  const [windSpeed, setWindSpeed] = useState(() => "6.1 kmph");
+  const [precipitation, setPrecipitation] = useState(() => "0.00 mm");
+  const [pressure, setPressure] = useState(() => "1025.0 mb");
+  // const [weatherStatus,setWeatherStatus] = useState(() => "Partly Cloudy")
+  const [weatherImage,setWeatherImage]  = useState(() => WeatherImages["partlyCloudy"])
+// const [countryList,getCountryList] = useState<CountryReport[]>(() => [])
+var countriesDetails:CountryReport[];
+  async function getWeatherDetails (){
+ 
+    countriesDetails =setCountryList()
+   console.log(countriesDetails);
+ 
+  
+  for(let i=0; i<countriesDetails.length;i++){
+    console.log(countriesDetails[i].countryName.toLowerCase()+":"+searchInput.toLowerCase());
+    
+    if(countriesDetails[i].cityName.toLowerCase() === searchInput.toLowerCase()){
```

Comment:
Instead of if else try to use include library function

Reply:
done